web/source/osk/bannerManager.ts

```diff
@@ -143,6 +143,8 @@ namespace com.keyman.osk {
             this.alwaysShow = optionSpec[key];
             break;
           case 'mayPredict':
+            // If this toggles our internal flag, it will generate events
+            // that reconfigures the banner (and internal engine state) appropriately.
             keyman.core.languageProcessor.mayPredict = optionSpec[key]
```

Comment:
Compare against https://github.com/keymanapp/keyman/blob/03c230c4e7c6e53320921ed91e15e11d8abe8fcc/common/core/web/input-processor/src/text/prediction/languageProcessor.ts#L407-L424

In particular:

https://github.com/keymanapp/keyman/blob/03c230c4e7c6e53320921ed91e15e11d8abe8fcc/common/core/web/input-processor/src/text/prediction/languageProcessor.ts#L420-L422

Also note this:

https://github.com/keymanapp/keyman/blob/03c230c4e7c6e53320921ed91e15e11d8abe8fcc/web/source/osk/bannerManager.ts#L67-L79

The removed `selectBanner` call is redundant and actually causes issues, as noted by @darcywong00's notes on #4961.  Fortunately, as it _is_ **redundant**, we can remove the issue-causing call.